### PR TITLE
[Add] Product detail page

### LINKED ISSUE #9: Product details page
Dependent on #4

### PR DESCRIPTION
**Pull request by gsc2001**
Fixes #9

### CODE REVIEW

**Comment by gsc2001** [comment]
Left work:
- [x] Carousel
- [x] Like review
- [x] Create review
- [ ] buy integration
- [ ] Add to cart integration

**Comment by gsc2001** [comment]
This has gone so big beyond my expectations. I am tired now 😓 
Marking this ready for review will do buy and cart integration in next issue

File: screens/ProductDetailScreen/Reviews.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// import React from 'react';
```
Comment (tichnas) [inline screens/ProductDetailScreen/Reviews.tsx:1]: This file??

**Comment by tichnas** [comment]
Remove the file and the comments. Approving this, so merge if you want to do these changes in next PR...

**Review by tichnas — APPROVED** [review]

File: screens/ProductDetailScreen/Reviews.tsx (hunk 1 above)
Comment (gsc2001) [reply screens/ProductDetailScreen/Reviews.tsx:1]: ok

**Comment by gsc2001** [comment]
Will remove it in next PR now

**Pull request merged**